src/style.css

```diff
@@ -2,0 +1,61 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+.body {
+  margin: 0;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  margin: 20px 8px 0;
+}
+
+.form-field {
+  display: none;
```

Comment:
The `.form-field` class has `display: none;` which makes the element invisible and unclickable. Make sure this is the intended behavior. If you want to hide the element only visually but keep it accessible for screen readers, you should use a different technique.